Source/DafnyCore/Options/CommonOptionBag.cs

```diff
@@ -349,6 +350,9 @@ public enum DefaultFunctionOpacityOptions {
 If verification fails, report a detailed counterexample for the first failing assertion (experimental).".TrimStart()) {
   };
 
+  public static readonly Option<string> BackendModuleName = new("--module-name",
+    @"If supported by the backend, this Option is used to create the top level module/project/package name".TrimStart()) {
```

Comment:
Could you move this option to the class `GoBackend`, since it's only used there? Also, I think you can leave out "If supported by the backend", since this option is only shown for backends that support it.

Reply:
I know at least Python needs to support that, and exactly why I have kept it in `CommonBagOption` . 

"If supported by the backend" => I had the same dilemma but if you think it's better to remove it, I'll remove it.

Reply:
> I know at least Python needs to support that, and exactly why I have kept it in CommonBagOption

How about we move it to a common location once it becomes commonly used? It'll be cheap to move it then, but it's hard to predict the future.

> I had the same dilemma but if you think it's better to remove it, I'll remove it.

Let's do that then.

Reply:
> How about we move it to a common location only once it becomes commonly used? It'll be cheap to move it then, but it's hard to predict the future.